app/src/full/java/io/homeassistant/companion/android/sensors/GeocodeSensorManager.kt

```diff
@@ -92,14 +92,19 @@ class GeocodeSensorManager : SensorManager {
                 )
             }.orEmpty()
 
-            val prettyAddress = address?.getAddressLine(0) ?: "Unknown"
+            var prettyAddressFound = true
```

Comment:
Do we need this variable since you check for null and set the unknown value already?

Reply:
No 👍  Resolved